elytron/src/main/java/org/wildfly/extension/elytron/TrivialResourceDefinition.java

```diff
@@ -37,10 +38,17 @@ final class TrivialResourceDefinition extends SimpleResourceDefinition {
     private final List<ResourceDefinition> children;
 
     private TrivialResourceDefinition(String pathKey, ResourceDescriptionResolver resourceDescriptionResolver, AbstractAddStepHandler add, AbstractRemoveStepHandler remove, AttributeDefinition[] attributes,
-            Map<AttributeDefinition, OperationStepHandler> readOnlyAttributes, Map<OperationDefinition, OperationStepHandler> operations, List<ResourceDefinition> children,
-            RuntimeCapability<?>[] runtimeCapabilities) {
+                                      Map<AttributeDefinition, OperationStepHandler> readOnlyAttributes, Map<OperationDefinition, OperationStepHandler> operations, List<ResourceDefinition> children,
+                                      RuntimeCapability<?>[] runtimeCapabilities) {
+
+        this(pathKey, resourceDescriptionResolver, add, remove, attributes, readOnlyAttributes, operations, children, runtimeCapabilities, Stability.DEFAULT);
+    }
+
+    private TrivialResourceDefinition(String pathKey, ResourceDescriptionResolver resourceDescriptionResolver, AbstractAddStepHandler add, AbstractRemoveStepHandler remove, AttributeDefinition[] attributes,
+                                      Map<AttributeDefinition, OperationStepHandler> readOnlyAttributes, Map<OperationDefinition, OperationStepHandler> operations, List<ResourceDefinition> children,
+                                      RuntimeCapability<?>[] runtimeCapabilities, Stability stability) {
```

Comment:
The above constructor is private, and the changes below mean that it will never be called.  Why not just change it rather than create a new one?

Reply:
@pferraro you're right, I changed the existing private constructor instead of creating new one